streams/cluster/src/main/java/com/hortonworks/streamline/streams/cluster/service/metadata/json/KafkaBrokerListeners.java

```diff
@@ -55,7 +65,19 @@ public ListenersPropParsed(List<ListenersPropEntry> parsedProps) {
 
         public ListenersPropParsed(ServiceConfiguration config) {
             try {
-                final Map<String, String> configMap = config.getConfigurationMap();
+                Map<String, String> confMap = config.getConfigurationMap();
```

Comment:
config.getConfigurationMap() throws IOException so we should add throw to method signature (bad to do that since only this constructor will throw IOE). Let's keep it as it is.